spec/index.html

```diff
@@ -319,49 +319,50 @@ <h3>RDF Vocabularies and Namespace IRIs</h3>
     <h3>Triple Terms and Reification</h3>
 
     <p>A <a>triple term</a> is an [=RDF triple=] used as an [=RDF term=]
-      in another triple. This use is a reference to a <a>proposition</a>. For
-      the triple term to also be asserted, it must [=appear=] in the same graph as both an <a>asserted
-      triple</a> and an unasserted <a>triple term</a>. This allows for statements to be made about statements
-      independent of their assertion within an <a>RDF graph</a>.
-    </p>
-
-    <p>[=RDF terms=] that [=appear=] in a [=triple term=] have the same
-      <a data-lt="denotes">denotation</a> as when they appear in an
-      [=asserted triple=] in the <a data-lt="RDF graph">graph</a>.
-      For this reason, we say that triple terms are
-      <dfn class="lint-ignore">transparent</dfn>.</p>
-
+      in another triple. 
+As an [=RDF triple=], it denotes a <a>proposition</a>.
     <p>
       A <dfn>reifying triple</dfn> is a triple where the <a>predicate</a> is <code>rdf:reifies</code>
       and the <a>object</a> is a <a>triple term</a>.
       The <a>subject</a> of that triple is called a <dfn>reifier</dfn>, and it can be the
       subject or object of other triples.
-      When the triple term of a <a>reifying triple</a> also [=appears=] in the
-      same graph as an <a>asserted triple</a>, the subset of triples that share
-      the same <a>reifier</a> as subject is called a <dfn>triple annotation</dfn>.
     </p>
-
     <p>
-      Concrete syntaxes, such as Turtle [[RDF12-TURTLE]], may have shortcuts
-      for specifying <a>reifying triples</a> and <a>triple annotations</a> more
-      succinctly.
+      A [=reifier=] may denote a variety of things that are related to the triple term's [=proposition=], 
+      such as a statement or belief that the [=proposition=] holds.
+      It is expected that the [=reifiers=] (rather than the [=triple terms=]) are used in further descriptions.
+    This section describes this common usage.
```

Comment:
```suggestion
      It is expected that the [=reifiers=] (rather than the [=triple terms=]) will be used in further descriptions.
      This section describes this common usage.
```